src/vxingest/main.py

```diff
@@ -383,36 +446,44 @@ def process_jobs(
 ) -> None:
     """
     Parses the given job docs with the appropriate method
```

Comment:
I'm not huge on having `process_jobs()` need to talk to couchbase. One of the things I tried to do was keep the functions that talk to couchbase in the main thread. Could this be refactored to something like:


```python
def process_jobs(
    cluster: Cluster,
    jobs: list[JobDoc] | list[JobSpec],
    startime: datetime,
    args,
    log_configurer: Callable,
) -> None:
"""
```